metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -219,7 +219,8 @@ def _handle_convert_column_evrs(  # noqa: C901 (complexity)
             elif exp == "expect_column_values_to_not_be_null":
                 column_profile.nullCount = res["unexpected_count"]
                 if "unexpected_percent" in res:
```

Comment:
I think the system can handle `column_profile.nullProportion = None`, so how about only setting it only when both of these are true. 

So I guess: 
```
if "unexpected_percent" in res and res["unexpected_percent"] is not None:
      column_profile.nullProportion = res["unexpected_percent"]
```

Reply:
That makes totally sense. I will try that!